test/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/HttpPostMultipart.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2am.micro.gw.tests.util;
+
+import javax.net.ssl.*;
+import java.io.*;
```

Comment:
Shall we remove this wild card import here?

Reply:
Fixed in 3b6cc6755810d748cd3de4fb25696a5692c08e78